packages/editor/src/components/entities-saved-states/style.scss

```diff
@@ -32,7 +32,7 @@
 }
 
 .entities-saved-states__post-meta {
-	margin-left: $grid-unit-30;
+	margin-left: 28px;
```

Comment:
Can we use an unit instead of hard-coding a margin?

Reply:
@cbravobernal In light of the [design changes](https://github.com/WordPress/gutenberg/pull/62542#issuecomment-2167528721), I now just use the preexisting `entities-saved-states__changes` class for styling.